cdap-cli/src/main/java/co/cask/cdap/cli/command/CreateDatasetInstanceCommand.java

```diff
@@ -52,21 +53,32 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     String datasetName = arguments.get(ArgumentName.NEW_DATASET.toString());
     String datasetPropertiesString = arguments.getOptional(ArgumentName.DATASET_PROPERTIES.toString(), "");
     String datasetDescription = arguments.getOptional(ArgumentName.DATASET_DESCRIPTON.toString(), null);
+    String datasetOwner = arguments.getOptional(ArgumentName.OWNER_PRINCIPAL.toString(), null);
```

Comment:
not sure I like this syntax. (that already applies to the description). We now have three optional arguments at the end of the command. So if only one is given, which one is it? It is ambiguous. We should instead use syntax to disambiguate. Such as: 
```
create dataset instance myDataset type someType [ with "a=1 b=2"] [ description "some thing" ] [ owner "alice/some.co@some.domain" ]. 
```
That would be unambiguous